src/main/java/edu/kis/powp/jobs2d/command/ComplexCommand.java

```diff
@@ -23,6 +25,19 @@ public void execute(Job2dDriver driver) {
         return commands.iterator();
     }
 
+    @Override
+    public void accept(Visitor visitor) {
+
+    }
```

Comment:
Why do you want to merge an empty method. It should be correctly implemented :). TIP: look how you did it in CompoundCommand class.